src/test/java/com/bnkrll/features/GetSessionsFlowTest.java

```diff
@@ -0,0 +1,54 @@
+package com.bnkrll.features;
+
+import com.bnkrll.model.Session;
+import com.bnkrll.service.SessionRepositorySimulator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+//import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Matchers.anyInt;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class GetSessionsFlowTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private SessionRepositorySimulator repository;
+
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
is this unused?

Reply:
Yes, I have removed the unused object from the test file